app/controllers/two-factor-auth-controller/get-index-controller.js

```diff
@@ -2,15 +2,9 @@
 
 // Local dependencies
 const {response} = require('../../utils/response.js')
-const paths = require('../../paths')
 
 module.exports = (req, res) => {
-  const PAGE_PARAMS = {
-    profile: paths.user.profile,
-    index: paths.user.twoFactorAuth.index
-  }
-
-  PAGE_PARAMS.authenticatorMethod = req.user.secondFactor
-
-  return response(req, res, 'twoFactorAuth/index', PAGE_PARAMS)
+  return response(req, res, 'twoFactorAuth/index', {
+    authenticatorMethod: req.user.secondFactor
```

Comment:
This is the critical change, we used to assume it was always `APP` now, it can be `APP || SMS`